README.md

```diff
@@ -140,3 +140,22 @@ Where `"name or id"` is a name or id from the `lead_providers` table.
 1. Get into Rails console for the environment you want to generate the token for.
 2. Run `EngageAndLearnApiToken.create_with_random_token!`
 3. Rails console should output a string, that's your unhashed token, you can keep it and use it to access E&L endpoints.
+
+## About the payment_calculator
+
+This engine performs all payment calculations for both [ECFs (Early Career Framework)](https://www.early-career-framework.education.gov.uk/) and [the reformed NPQs (National Professional Qualification)](https://www.gov.uk/government/publications/national-professional-qualifications-frameworks-from-september-2021) so that training providers can be paid the correct amount.
```

Comment:
> This engine

What engine? Can you point to it?

Reply:
Good point, that's rather ambiguous. Iterated.